docs/src/_redirects

```diff
@@ -0,0 +1 @@
+/ https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/vue/
```

Comment:
Because of this redirection, the v1 website won't be easily accessible via the URL (https://community.algolia.com/vue-instantsearch/ points to Algolia).

Do we want to create the endpoint `https://community.algolia.com/vue-instantsearch/v1`? This is what we do for [InstantSearch.js v2](https://community.algolia.com/instantsearch.js/v2).

Reply:
that's a possibility, but would break all URLs. What I did for now is change the URL we use to the getting started URL and thus keep all existing URLs. Maybe some kind of redirection of all things to v1 would be possible as you said. 🤔